index.d.ts

```diff
@@ -120,26 +120,26 @@ export interface AxiosInterceptorManager<V> {
   eject(id: number): void;
 }
 
-export interface AxiosInstance {
-  (config: AxiosRequestConfig): AxiosPromise;
-  (url: string, config?: AxiosRequestConfig): AxiosPromise;
-  defaults: AxiosRequestConfig;
+export interface AxiosInstance<C extends AxiosRequestConfig=AxiosRequestConfig> {
```

Comment:
better to call this something like `TConfig` or just `T` for convention

Reply:
Yes, maybe. I think that `C` would work well for Config, below we already have `T` for the expected response type, `R` for the Axios response